codec/enforceproto/testdata/svc.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package service;
```

Comment:
I think this code can just simply be in `codec/testdata/proto.proto`.

Reply:
Ack, thank you! I didn't know where to put it but you gave me the direction.